<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeWidget" name="treeWidget">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <column>
        <property name="text">
         <string>Objects</string>
        </property>
       </column>
      </widget>
      <widget class="QGraphicsView" name="graphicsView"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>Mode</string>
    </property>
    <addaction name="actionEdit_Mode"/>
    <addaction name="actionSimulate_Mode"/>
   </widget>
   <widget class="QMenu" name="menuSimulating">
    <property name="title">
     <string>Simulating</string>
    </property>
    <addaction name="actionSimulate_All"/>
    <addaction name="actionStopStartSimulating"/>
   </widget>
   <widget class="QMenu" name="menuGame_Test">
    <property name="title">
     <string>Game Test</string>
    </property>
    <addaction name="actionShoot_Mode"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>view</string>
    </property>
    <addaction name="actionZoom_in"/>
    <addaction name="actionZoom_out"/>
   </widget>
   <addaction name="menuMode"/>
   <addaction name="menuSimulating"/>
   <addaction name="menuGame_Test"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStopStartSimulating"/>
   <addaction name="actionZoom_in"/>
   <addaction name="actionZoom_out"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSimulate_All">
   <property name="text">
    <string>Simulate All</string>
   </property>
  </action>
  <action name="actionStopStartSimulating">
   <property name="icon">
    <iconset resource="rc.qrc">
     <normaloff>:/zoom/icon/start.png</normaloff>:/zoom/icon/start.png</iconset>
   </property>
   <property name="text">
    <string>StopStartSimulating</string>
   </property>
  </action>
  <action name="actionEdit_Mode">
   <property name="text">
    <string>Edit Mode</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="actionSimulate_Mode">
   <property name="text">
    <string>Simulate Mode</string>
   </property>
  </action>
  <action name="actionShoot_Mode">
   <property name="text">
    <string>Shoot Mode</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="icon">
    <iconset resource="rc.qrc">
     <normaloff>:/zoom/icon/zoom_in.png</normaloff>:/zoom/icon/zoom_in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom in</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="icon">
    <iconset resource="rc.qrc">
     <normaloff>:/zoom/icon/zoom_out.png</normaloff>:/zoom/icon/zoom_out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom out</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="rc.qrc"/>
 </resources>
 <connections/>
</ui>
